packages/babel-plugin-proposal-class-properties/test/fixtures/private-loose/native-classes/output.js

```diff
@@ -0,0 +1,27 @@
+class Foo {
+  constructor() {
+    Object.defineProperty(this, _bar, {
+      writable: true,
+      value: "bar"
+    });
+  }
+
+  static test() {
+    return babelHelpers.classStaticPrivateFieldLooseBase(Foo, Foo)._foo;
+  }
+
+  test() {
+    return babelHelpers.classPrivateFieldLooseBase(this, _bar)[_bar];
+  }
+
+}
+
+Foo._foo = "foo";
```

Comment:
My last nit: This should be using `Object.defineProperty` with `{ enumerable: false, configurable: false }`. We can do this in a follow PR if you'd like.

Reply:
That is a good point, I'll try to put that here before we merge